libs/shared/ui/src/lib/components/inputs/input-text/input-text.tsx

```diff
@@ -31,6 +32,11 @@ export function InputText(props: InputTextProps) {
 
   const isDisabled = disabled ? 'input--disabled !border-element-light-lighter-500' : ''
 
+  const displayPicker = () => {
+    const input: any = inputRef.current?.querySelector('input')
```

Comment:
We can't type this?

Reply:
The problem is actually if I type the input, I have an error on the `showPicker` function. I tried to avoid it but I can't find any solution